producers-api/build.gradle

```diff
@@ -28,3 +28,73 @@ bootJar {
 test {
     useJUnitPlatform()
 }
+
+ext.cxfVersion = "3.5.4"
```

Comment:
Is there any way the cxfVersion variable from the main build.gradle file can be used?  This would reduce the risk of there being a mis-match in version numbers between the two.

Reply:
Variable has been removed from file so that one from main build.gradle is used.